resources/css/main.css

```diff
@@ -298,6 +298,10 @@ dl dt {
     height: 50px;
 }
 
+#navigation a, #navigation a:link {
```

Comment:
Might want to add a background color and border to make it easier to see:
![image](https://user-images.githubusercontent.com/1960707/140086576-072d7162-322e-4249-9165-6f52962b38dc.png)

Reply:
That would require (in the ideal case) copying the header color, which isn't great. I reworked the positioning of the links instead.